Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.yaml

```diff
@@ -2,39 +2,39 @@ apiVersion: "sparkoperator.hpe.com/v1beta2"
 kind: SparkApplication
 metadata:
   name: spark-fts-{{ts_nodash|replace("T", "")}}
+  namespace: spark
 spec:
   sparkConf:
-    spark.mapr.extraconf.secret: {{dag_run.conf.get("s3_secret_name", "spark-s3-creds")}}
     spark.hadoop.fs.s3a.path.style.access: "true"
     spark.hadoop.fs.s3a.impl: "org.apache.hadoop.fs.s3a.S3AFileSystem"
     spark.hadoop.fs.s3a.connection.ssl.enabled: "true"
+    spark.hadoop.fs.s3a.access.key: <access-token>
+    spark.hadoop.fs.s3a.secret.key: <secret-key>
```

Comment:
It will not work. There are two ways: we add a new parameter for JWT token which we are passing in the params into this yaml. Or we continue to use the secret with s3 credentials.